test/typings/either-spec.ts

```diff
@@ -57,7 +57,7 @@ log(unpacked).run();
 log2(getMessage(null).flatMap(getType)).performUnsafeIO();
 log3(getMessage({type: 'x', payload: null}).chain(getType));
 
-const nameError: string = Left('-- Adamovisch').leftMap(n => n.split(' ').shift()).left();
+const nameError: string = Left('-- Adamovisch').leftMap((n:string) => n.split(' ').shift() || "").left();
```

Comment:
Yeah I use `noImplicitAny` in commercial apps :) (along with `no-any` tslint rule ;) )

I see why it doesn't work here:

```typescript
interface ILeftStatic extends IMonadFactory {
  <F, V>(val: F): Either<F, V>;
}
```

And as it needs 2 type parameters it doesn't handle inference of one `Either` side properly :( Not nice…

So leave it as is.

Just please run `npm run publ` and commit changes (as we keep build in repo during *Alpha* phase for ease of debugging).